src/linker/wasm.rs

```diff
@@ -64,27 +61,27 @@ pub fn link(input: &[u8], name: &str) -> Vec<u8> {
         parity_wasm::deserialize_buffer(&output).expect("cannot deserialize llvm wasm");
 
     {
-        let imports = module.import_section_mut().unwrap().entries_mut();
-        let mut ind = 0;
-
-        while ind < imports.len() {
-            if imports[ind].field().starts_with("seal") {
-                let module_name = match imports[ind].field() {
-                    "seal_set_storage" => "seal2",
-                    "seal_clear_storage"
-                    | "seal_contains_storage"
-                    | "seal_get_storage"
-                    | "seal_instantiate"
-                    | "seal_terminate"
-                    | "seal_random"
-                    | "seal_call" => "seal1",
-                    _ => "seal0",
-                };
-                *imports[ind].module_mut() = module_name.to_owned();
-            }
-
-            ind += 1;
-        }
+        // let imports = module.import_section_mut().unwrap().entries_mut();
+        // let mut ind = 0;
+
+        // while ind < imports.len() {
+        //     if imports[ind].field().starts_with("seal") {
+        //         let module_name = match imports[ind].field() {
+        //             "seal_set_storage" => "seal2",
+        //             "seal_clear_storage"
+        //             | "seal_contains_storage"
+        //             | "seal_get_storage"
+        //             | "seal_instantiate"
+        //             | "seal_terminate"
+        //             | "seal_random"
+        //             | "seal_call" => "seal1",
+        //             _ => "seal0",
+        //         };
+        //         *imports[ind].module_mut() = module_name.to_owned();
+        //     }
+
+        //     ind += 1;
+        // }
```

Comment:
This change can be ignored. Because Soroban has no always imported value, if a contract doesn't use any host functions/imports, there is no import section to mutate and line 64 panicked. I'm sure there would be a better way to code this to just check if the import section existed or not.